input/otlp/traces.go

```diff
@@ -270,7 +270,19 @@ func TranslateTransaction(
 		k := replaceDots(kDots)
 		switch v.Type() {
 		case pcommon.ValueTypeSlice:
-			setLabel(k, event, ifaceAttributeValue(v))
+			switch kDots {
+			case "elastic.profiler_stack_trace_ids":
+				var vSlice = v.Slice()
+				event.Transaction.ProfilerStackTraceIds = make([]string, 0, vSlice.Len())
+				for i := 0; i < vSlice.Len(); i++ {
```

Comment:
use [`range`](https://gobyexample.com/range) instead?

Reply:
imo this is specific to OTel and we can't range over it.